tests/test_efs.py

```diff
@@ -51,6 +54,20 @@ def test_mount_target_loading(self):
         resources = p.run()
         self.assertEqual(len(resources), 2)
 
+    def test_mount_target_security_group(self):
+        factory = self.replay_flight_data('test_efs_mount_secgroup')
+        p = self.load_policy({
+            'name': 'test-mount-secgroup',
+            'resource': 'efs-mount-target',
+            'filters': [
+                {'type': 'security-group',
```

Comment:
to exercise the cache usage, we need to have two security group filters in a policy.

Reply:
Can you approve as-is, and we track an action item to enhance the tests by adding the 2nd security-group?  I've got so many must do's, I'm not likely to get to this until March...

Reply:
i'll work on it.